components/tidb_query_vec_expr/src/impl_string.rs

```diff
@@ -1944,6 +1979,112 @@ mod tests {
         }
     }
 
+    #[test]
+    fn test_make_set() {
```

Comment:
Could you also test the case when the first parameter is `NULL`?

Reply:
Third test case have the first parameter equal to `NULL`

Reply:
Ah sorry, I miss it.